src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -205,3 +210,14 @@ export const hasMonthYear = (err, fieldData) => {
     err.addError('Please provide both month and year');
   }
 };
+
+export const validateDisabilityName = (err, fieldData) => {
+  if (
+    !LOWERED_DISABILITY_DESCRIPTIONS.includes(fieldData.toLowerCase()) &&
+    !VDC_DISABILITY_NAME_REGEX.test(fieldData)
+  ) {
+    // technically single quotes (’) are allowed as well but leaving out of
+    // this message to avoid confusing veterans who can't tell the difference
```

Comment:
TBD what we do with this, but if we end up wanting to allow apostrophes and transform them to single quotes in the submit transformer, I'd rather submit this as-is and then push up another PR for that functionality separately.

Reply:
Yeah, this sounds fine to me.